src/dataviews/dataview-model-base.js

```diff
@@ -171,22 +159,11 @@ module.exports = Model.extend({
     });
   },
 
-  _onSourceChanged: function (model) {
-    var changedKeys = model && model.changed
-      ? _.keys(model.changed)
-      : '';
-    track(new Error('[SOURCE] _onSourceChanged [' + changedKeys + ']'));
-  },
-
   _setupAnalysisStatusEvents: function () {
-    this._removeExistingAnalysisBindings();
-    if (this.getSource()) {
-      this.getSource().on('change:status', this._onAnalysisStatusChange, this);
-    }
+    this.getSource().on('change:status', this._onAnalysisStatusChange, this);
```

Comment:
This function `_setupAnalysisStatusEvents` is not called ever.

You should add this logic to `initBinds` in order to get the dataviews to react to analysis status changes.

Reply:
It's actually being called in the `initialized` method, but I'll include this logic in `initBinds`. 👍